lib/model/story.ts

```diff
@@ -67,3 +98,17 @@ export function fixTitle(title = '') {
     .replace(new RegExp(QUOTES.doubleLeft, 'g'), QUOTES.singleLeft)
     .replace(new RegExp(QUOTES.doubleRight, 'g'), QUOTES.singleRight)
 }
+
+export const ADMIN_INCLUDE = {
+  postalCode: {
+    include: {
+      ridings: {
+        include: {
+          riding: true,
+        },
+      },
+    },
+  },
+}
+
+export type AdminStory = Prisma.StoryGetPayload<{ include: typeof ADMIN_INCLUDE }>
```

Comment:
The queries for stories in the admin context need this `include` clause to follow relations through the postal code to the riding, so we also need a new `AdminStory` type that includes all that.